src/libs/actions/Report.js

```diff
@@ -1242,7 +1242,7 @@ Onyx.connect({
  */
 function editReportComment(reportID, originalReportAction, textForNewComment) {
     const parser = new ExpensiMark();
-    const htmlForNewComment = parser.replace(textForNewComment);
+    const htmlForNewComment = parser.replace(textForNewComment, {filterRules: _.map(_.filter(parser.rules, (({name}) => !['autolink'].includes(name))), ({name}) => name)});
```

Comment:
```suggestion
    const htmlForNewComment = parser.replace(textForNewComment, { filterRules: _.filter(_.pluck(parser.rules, 'name'), name => name !== 'autolink')});
```
Can we simplify the expression?

Reply:
I agree with simplifying the expression! Thanks for the suggestion @mananjadhav!

Reply:
Handled.